SpringBoot-Board/src/main/java/com/homework/springbootboard/model/Post.java

```diff
@@ -0,0 +1,64 @@
+package com.homework.springbootboard.model;
+
+import com.homework.springbootboard.dto.PostDto;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Entity
+@Table(name = "post")
+public class Post extends BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @Column(name = "title", nullable = false, length = 100)
+    private String title;
+
+    @Lob
+    @Column(name = "content", nullable = false)
+    private String content;
+
+    @ManyToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+    @JoinColumn(name = "user_id", referencedColumnName = "id")
+    private User user;
```

Comment:
데이터가 없어서 null인지 의도적으로 null로 선언한 것인지 구분할 수가 없기때문에 List 초기화를 해주는 게 좋다고 해유

```suggestion
    @JoinColumn(name = "user_id", referencedColumnName = "id")
    private User user = new ArrayList<>();
```

Reply:
아하... 부희님 덕분에 또 하나 배웠습니다!! 감사합니다 ㅎㅎ

Reply:
엇 제가 리뷰를 잘못 남겼네요😅 넹 수형님이 말씀하신 것처럼 User의 List<Post>부분을 초기화해야합니닷!!